.travis.yaml

```diff
@@ -0,0 +1,7 @@
+language: python
+python:
+  - "3.7"
+  - "3.8"
+# command to run tests
+script:
+  - pytest
```

Comment:
The command we have been using is `python -m pytest`

I'm not sure if this should be replaced by this or not